docs/docs/core/09-telemetry.md

```diff
@@ -79,50 +79,51 @@ The following examples expose too much cardinality and may not even prove to be
+| `ibc_client_update`             | Total number of client updates                               | update          | counter |
+| `ibc_client_upgrade`            | Total number of client upgrades                              | upgrade         | counter |
+| `ibc_client_misbehaviour`       | Total number of client misbehaviours                         | misbehaviour    | counter |
+| `ibc_connection_open-init`      | Total number of connection `OpenInit` handshakes             | handshake       | counter |
+| `ibc_connection_open-try`       | Total number of connection `OpenTry` handshakes              | handshake       | counter |
+| `ibc_connection_open-ack`       | Total number of connection `OpenAck` handshakes              | handshake       | counter |
+| `ibc_connection_open-confirm`   | Total number of connection `OpenConfirm` handshakes          | handshake       | counter |
+| `ibc_channel_open-init`         | Total number of channel `OpenInit` handshakes                | handshake       | counter |
+| `ibc_channel_open-try`          | Total number of channel `OpenTry` handshakes                 | handshake       | counter |
+| `ibc_channel_open-ack`          | Total number of channel `OpenAck` handshakes                 | handshake       | counter |
+| `ibc_channel_open-confirm`      | Total number of channel `OpenConfirm` handshakes             | handshake       | counter |
+| `ibc_channel_close-init`        | Total number of channel `CloseInit` handshakes               | handshake       | counter |
+| `ibc_channel_close-confirm`     | Total number of channel `CloseConfirm` handshakes            | handshake       | counter |
+| `tx_msg_ibc_recv_packet`        | Total number of IBC packets received                         | packet          | counter |
+| `tx_msg_ibc_acknowledge_packet` | Total number of IBC packets acknowledged                     | acknowledgement | counter |
+| `ibc_timeout_packet`            | Total number of IBC timeout packets                          | timeout         | counter |
+| `store_iavl_get`                | Duration of an IAVL `Store#Get` call                         | ms              | summary |
+| `store_iavl_set`                | Duration of an IAVL `Store#Set` call                         | ms              | summary |
+| `store_iavl_has`                | Duration of an IAVL `Store#Has` call                         | ms              | summary |
+| `store_iavl_delete`             | Duration of an IAVL `Store#Delete` call                      | ms              | summary |
+| `store_iavl_commit`             | Duration of an IAVL `Store#Commit` call                      | ms              | summary |
+| `store_iavl_query`              | Duration of an IAVL `Store#Query` call                       | ms              | summary |
+| `blockdelivery`                 | Duration of deliver a whole block                            | ms              | summary |
```

Comment:
this already exists within in tendermint: https://github.com/cometbft/cometbft/blob/main/docs/core/metrics.md. Is there a reason it should be added to the sdk as well?

Reply:
nice, just not familiar with them, closing now

Reply:
sweet, happy to add here as well if the data the tendermint metrics provide isn't sufficient

Reply:
the `state_block_processing_time` only count from begin block to end block, not commit step though.